distributed/worker.py

```diff
@@ -3339,10 +3339,10 @@ def release_key(
             self._notify_plugins(
                 "release_key", key, state_before, cause, reason, report
             )
-        except CommClosedError:
+        except CommClosedError:  # pragma: no cover
```

Comment:
Batched send close is something we're not properly testing but it is known to cause instability. Leaving this in

```suggestion
        except CommClosedError:
```